openedx/core/djangoapps/user_api/views.py

```diff
@@ -65,24 +65,32 @@ def get(self, request):
 
         """
         form_desc = FormDescription("post", reverse("user_api_login_session"))
+        email_or_username = settings.FEATURES.get('ENABLE_LOGIN_BY_EMAIL_OR_USERNAME', False)
 
         # Translators: This label appears above a field on the login form
         # meant to hold the user's email address.
         email_label = _(u"Email")
+        if email_or_username:
+            email_label = _(u"Email or username")
 
         # Translators: This example email address is used as a placeholder in
         # a field on the login form meant to hold the user's email address.
         email_placeholder = _(u"username@domain.com")
 
         # Translators: These instructions appear on the login form, immediately
         # below a field meant to hold the user's email address.
-        email_instructions = _("The email address you used to register with {platform_name}").format(
+        message = _("The email address you used "
+                    "to register with {platform_name}")
+        if email_or_username:
+            message = _("The email address or username you used "
+                        "to register with {platform_name}")
+        email_instructions = message.format(
```

Comment:
We should also update the variable here to be `account_id_instructions`.
Basically, everywhere the word `email` is used in variable, let's consider updating it to reflect it maybe showing a username as well.